runtime/error.c

```diff
@@ -181,3 +187,83 @@ TaggedPtr BAL_LANG_ERROR_NAME(message)(TaggedPtr error) {
     ErrorPtr ep = taggedToPtr(error);
     return ep->message;
 }
+
+void _bal_print_mangled_name(const char *mangledName, FILE *fp) {
+    // If mangled name starts from _B_, the identifier is non-public
+    if (mangledName[2] == '_') {
+        fprintf(fp, "%s", mangledName + 3);
+        return;
+    }
+    // The org-name is not considered as a part of function name in backtrace.
+    int initialOffset = getOrgNameOffset(mangledName);
+
+    // Make sure parsing starts from the first character of
+    // nqual or local-name or mod-name
+    bool isFirstQualifier = true;
+    printModuleName(mangledName + initialOffset, &isFirstQualifier, fp);
+}
+
+static int getOrgNameOffset(const char *mangledName) {
+    // Offset starts from '_B'
+    int offset = 2;
+    if (mangledName[offset] == 'b') {
+        offset++;
+    }
+    return offset + getzQualifierOffset(mangledName + offset);
+}
+
+static int getzQualifierOffset(const char *mangledName) {
```

Comment:
Why `z` in `getz` ?

Reply:
Actually I am using the keywords used in https://github.com/ballerina-platform/nballerina/issues/448#issuecomment-919844131

Reply:
Correct camlCase encoding of that would be `getZQualifierOffset`.

Reply:
Changed